toqito/states/ghz.py

```diff
@@ -91,7 +91,7 @@ def ghz(dim: int, num_qubits: int, coeff: list[int] = None) -> sparse:
     for i in range(1, num_qubits):
         dim_sum += dim**i
 
-    ret_ghz_state = sparse.lil_matrix((dim**num_qubits, 1))
+    ret_ghz_state = sparse.lil_array((dim**num_qubits, 1))
     for i in range(1, dim + 1):
```

Comment:
A general comment about the return type at L7 (I can't make a comment directly on that line)

Is the return type of this function a `sparse.lil_array`? I think you could edit that line to add this detail.

Reply:
Yes, updated the line to `sparse.lil_array`.